monitor/sth_fetcher_test.go

```diff
@@ -38,6 +38,13 @@ func TestObserveSTH(t *testing.T) {
 	// Make an STH observation
 	m.fetcher.observeSTH()
 
+    // STH fetches are counted
```

Comment:
I have no idea why golangcibot is triggering on this. Seems like a bug.

Reply:
Oh, I saw Kiel's PR. My guess: goimports probably `gofmts` too, and this check may be running before `gofmt` (if that even runs at all). So it is confusing because it's really a formatting problem, not an import problem.

Reply:
Done! Thanks @ezekiel.